cryptostore/data/parquet.py

```diff
@@ -70,10 +70,13 @@ def aggregate(self, data):
         cols = {name: [] for name in names}
 
         for entry in data:
-            for key in entry:
+            for key in names:
```

Comment:
why did this change from entry to name?

Reply:
Hi @bmosco,
This came from my previous change: I was discarding `feed`and `pair`columns from `names`.
To avoid having them in the array list, I was thus using directly `names`.
Now that I have reverted the change to keep them following your first feedback, yes, I will also revert this change, no worried. I will do this this evening.
Bests

Reply:
@bmoscon ok, change reverted. Sorry for the miss.